pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -939,20 +939,34 @@ type CloudSpec struct {
 	// "aws", then the `aws` field must be set).
 	ProviderName string `json:"providerName"`
 
-	Fake                *FakeCloudSpec                `json:"fake,omitempty"`
-	Digitalocean        *DigitaloceanCloudSpec        `json:"digitalocean,omitempty"`
-	BringYourOwn        *BringYourOwnCloudSpec        `json:"bringyourown,omitempty"`
-	AWS                 *AWSCloudSpec                 `json:"aws,omitempty"`
-	Azure               *AzureCloudSpec               `json:"azure,omitempty"`
-	Openstack           *OpenstackCloudSpec           `json:"openstack,omitempty"`
-	Packet              *PacketCloudSpec              `json:"packet,omitempty"`
-	Hetzner             *HetznerCloudSpec             `json:"hetzner,omitempty"`
-	VSphere             *VSphereCloudSpec             `json:"vsphere,omitempty"`
-	GCP                 *GCPCloudSpec                 `json:"gcp,omitempty"`
-	Kubevirt            *KubevirtCloudSpec            `json:"kubevirt,omitempty"`
-	Alibaba             *AlibabaCloudSpec             `json:"alibaba,omitempty"`
-	Anexia              *AnexiaCloudSpec              `json:"anexia,omitempty"`
-	Nutanix             *NutanixCloudSpec             `json:"nutanix,omitempty"`
+	Fake *FakeCloudSpec `json:"fake,omitempty"`
+	// Digitalocean specifies access data to DigitalOcean.
```

Comment:
"access data" is not the right description. These fields, not just for digitalocean, do not just contain credentials (in fact, in most cases they will not contain those).

These fields contain something like "cloud provider configuration".

Reply:
I tried to obtain and match here with the actual description available for each cloud provider spec, similar to what I see is done in most of the CRDs. For example [DigitialOceanCloudSpec](https://docs.kubermatic.com/kubermatic/main/references/crds/#digitaloceancloudspec). 
But I am with you, we can make it better as per suggestion. Will fix it for all, thanks!